### PR TITLE
Profile update

### PR DESCRIPTION
**Pull request by njmulsqb**

### CODE REVIEW

**Review by kingthorin — APPROVED** [review]

**Review by psiinon — APPROVED** [review]

File: site/content/docs/desktop/credits.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ People who have made contributions to ZAP over the years, in alphabetical order:
 |   | Guifre Ruiz Utges                                                                                                     |
 |   | Dobin Rutishauser - Compass Security AG                                                                               |
 |   | Bunyanuch Saengnet                                                                                                    |
-|   | Najam Ul Saqib ([@njmulsqb](https://www.twitter.com/njmulsqb))                                                        |
+|   | Najam Ul Saqib ([@njmulsqb](https://www.infosec.exchange/@cybersoldier))                                              |
```
Comment (psiinon) [inline site/content/docs/desktop/credits.md:157]: This file is actually autogenerated, so will be overwritten at the next release.
You need to also update https://github.com/zaproxy/zap-core-help/blob/main/addOns/help/src/main/javahelp/contents/credits.html#L176

Comment (njmulsqb) [reply site/content/docs/desktop/credits.md:157]: Alright, is there anyway for me to remove Twitter URL from my page in team section?

Comment (psiinon) [reply site/content/docs/desktop/credits.md:157]: Of course. For historical reasons thats here https://github.com/zaproxy/zaproxy-website/blob/800767b7d04f604a6dc7c71a7284fab5ed48ba03/site/data/authors.yml#L117

Comment (njmulsqb) [reply site/content/docs/desktop/credits.md:157]: Yeah I removed this line and then it was just showing the URL `https://twitter.com` on my profile which is strange.

Comment (psiinon) [reply site/content/docs/desktop/credits.md:157]: Not in this PR you havnt 😛 
There are currently no changes to the authors file that I can see...

Comment (psiinon) [reply site/content/docs/desktop/credits.md:157]: Ah, thats a bug in the template then - I'll have a look at that at some point...

Comment (njmulsqb) [reply site/content/docs/desktop/credits.md:157]: May I add this as an issue here?

Comment (thc202) [reply site/content/docs/desktop/credits.md:157]: Done in #2152.

File: site/content/docs/desktop/credits.md [hunk 2]
```diff
@@ -154,7 +154,7 @@ People who have made contributions to ZAP over the years, in alphabetical order:
 |   | Guifre Ruiz Utges                                                                                                     |
 |   | Dobin Rutishauser - Compass Security AG                                                                               |
 |   | Bunyanuch Saengnet                                                                                                    |
-|   | Najam Ul Saqib ([@njmulsqb](https://www.twitter.com/njmulsqb))                                                        |
+|   | Najam Ul Saqib ([@njmulsqb](https://www.twitter.com/njmulsqb))                                              |
```
Comment (thc202) [inline site/content/docs/desktop/credits.md:157]: This should have been properly reverted.

**Pull request merged**